api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -263,4 +266,19 @@ public CohortDefinition getTxCurrCompositionCohort(String cohortName, CohortDefi
 		TxCurrComposition.setCompositionString(compositionString);
 		return TxCurrComposition;
 	}
+	
+	@DocumentedDefinition(value = "getTxCurrConcept1255Cohort")
+	public CohortDefinition getTxCurrConcept1255Cohort() {
+		CodedObsCohortDefinition definition = new CodedObsCohortDefinition();
+		definition.setName("CONCEPT1255");
+		definition.addParameter(new Parameter("onOrBefore", "onOrBefore", Date.class));
+		definition.addParameter(new Parameter("location", "location", Location.class));
```

Comment:
The field name for location on `CodedObsCohortDefinition` is `locationList`, but we can't remove `location` because it's used by other cohorts on the same indicator.

Reply:
> Another thing: how are you testing this query in isolation to the other queries in the query combination?

I'm changing the Java code to use only this cohort for one of the colums.

Reply:
Ok so there should only be one parameter for `location`. They are infact the same just different labels. Please change to just use `location`.